### PR TITLE
Logger functions init and watch scripts

### PR DESCRIPTION
**Pull request by cashd**
- Started the logger functions which will standardize our output and make it more readable
- Created the watch scripts which saves evm_snapshot after every block and can revert N blocks. useful for testing frontend flows.

### CODE REVIEW

File: apps/core-v2-testnet/tsconfig.json [hunk 1]
```diff
@@ -6,8 +6,12 @@
     "outDir": "dist",
     "forceConsistentCasingInFileNames": true,
     "strict": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "baseUrl": ".",
+    "paths": {
+      "@utils/*": ["utils/*"]
```
Comment (DannyDelott) [inline apps/core-v2-testnet/tsconfig.json:12]: Do we need this? Seems like we should just use `rootDir` and avoid the path aliases here.

Comment (cashd) [reply apps/core-v2-testnet/tsconfig.json]: yeah we don't need this just removed it!

File: apps/core-v2-testnet/.eslintrc.js [hunk 2]
```diff
@@ -15,7 +20,10 @@ module.exports = {
       "error",
       { ignores: ["modules"] },
     ],
+    // disabling b/c this node rule has problems with ts paths, added import plugin for coverage instead
+    "node/no-missing-import": "off",
```
Comment (DannyDelott) [inline apps/core-v2-testnet/.eslintrc.js:24]: nice comment 👍

File: apps/core-v2-testnet/.eslintrc.js [hunk 3]
```diff
@@ -15,7 +20,10 @@ module.exports = {
...
     // disabling this rule because the testnet will need to output to log frequently
     "no-console": "off",
+    "no-process-exit": "off",
```
Comment (DannyDelott) [inline apps/core-v2-testnet/.eslintrc.js:27]: add comment?

File: apps/core-v2-testnet/scripts/watch.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  // stack to store snapshots
+  let stack: SnapshotRestorer[] = [];
+
+  // register event listener to new blocks
+  provider.on("block", async (block: number) => {
+    const snapshot = await takeSnapshot();
+    stack.push(snapshot);
+  });
+
+  await forever(async () => {
```
Comment (DannyDelott) [inline apps/core-v2-testnet/scripts/watch.ts:27]: minor: if we can, let's switch this to just `setInterval(cb, 0)`. Simpler I think, and gives us the ability to stop it w/ `clearInterval` if we wanted.

Comment (cashd) [reply apps/core-v2-testnet/scripts/watch.ts:27]: this runs unbounded, since a new stream is getting created every time the interval is called

**Review by DannyDelott — APPROVED** [review]

**Pull request merged**